torch/_higher_order_ops/utils.py

```diff
@@ -313,16 +313,23 @@ def fw_with_masks(*args):
 # In PyTorch there is the convention that None gradients are
```

Comment:
This is not a convention i think. More like "None" is a an optimization for zero grad and since None is difficult to handle in cuda graph, we don't do this optimizaiton

Reply:
I changed the comment